src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -41,7 +41,9 @@ public enum IamAction {
   // journal
   VIEW_JOURNAL,
   VIEW_SNAPSHOT_BUILDER_SETTINGS,
-  UPDATE_SNAPSHOT_BUILDER_SETTINGS;
+  UPDATE_SNAPSHOT_BUILDER_SETTINGS,
+  RUN_UPGRADE_FLIGHT,
+  REGISTER_DRS_ALIASES;
```

Comment:
Would it be worth bundling the admin-only actions together?  At any rate, this comment is somewhat out of date:
```
  /* CONFIGURE is only available to admin users so can be used to check for admin access to TDR */
  CONFIGURE,
```